testtools/util.go

```diff
@@ -286,7 +286,7 @@ func CreateWalPageWithContinuation() []byte {
 		Info:             walparser.XlpFirstIsContRecord,
 		RemainingDataLen: 12312,
 	}
-	data := make([]byte, 20)
```

Comment:
This line was correct, after defining data we call `binary.LittleEndian.PutUint16` that needs byte slice with non-zero length. If any linter complains, you can disable it by putting `// nolint name`

Reply:
fixed